backend/handleError.ts

```diff
@@ -12,13 +12,17 @@ export function handleError(res: NextApiResponse, e: unknown) {
 
   if (e instanceof PrismaClientKnownRequestError) {
     if (e.code === 'P2025') {
-      return res.status(404).send('Gift was not found on the server!');
+      if (e.meta?.modelName === 'Gift') {
+        return res.status(404).send('Gift was not found on the server!');
+      }
+      return res.status(404).send('Record was not found on the server!');
```

Comment:
Tekisin tästä geneerisen, joka toimii kaikilla modeleilla:
```suggestion
      const modelName =
        typeof e.meta?.modelName === 'string' ? e.meta?.modelName : 'Record';
      return res.status(404).send(modelName + ' was not found on the server!');
```

Reply:
Vau! Tämä oli hyvä, kiitos!! Ei käynyt edes mielessä, vaikka hyvin simppeli tapa onkin :smile: